builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -521,7 +521,12 @@ def updateSwaggerConfig() {
 			sh "sed -i -- 's/{domain}/${service_config['domain']}/g' swagger/swagger.json"
 
 			def region = "${configLoader.AWS.REGION}"
-			def role = "${configLoader.AWS.ROLEID}"
+      def role
+      if (service_config['domain'] && service_config['domain'] == "jazz") {
```

Comment:
@SiniWilson  @rajeevr2715  Shouldn't the role come from service metadata (or service_config)? Aren't we persisting role information in service catalog?

Reply:
Done.